configure.ac

```diff
@@ -86,6 +86,9 @@ else
 fi
 AM_CONDITIONAL([HAVE_HELP2MAN], [test x$HELP2MAN = xhelp2man])
 
+# Only run parts of the tests if we are cross compiling
+AM_CONDITIONAL([CROSS_COMPILING], [test "x$cross_compiling" = xyes])
```

Comment:
Well, I guess you're right. If I were to cross-compile for arm the test scripts would probably work. We should probably add a note in the Makefile that technically the scripts work in the face of cross-compilation, as long as the executables have no suffix and do not require an emulator such as wine